### PR TITLE
Integrate search

### PR DESCRIPTION
**Pull request by vileyne**

### CODE REVIEW

File: code/lightna/magento-os-frontend/js/common/Search.js [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+        this.search = $('#search');
+        this.searchWrap = $('.search__wrap');
+        this.clearAction = $(this.actions.clear);
+        this.bindEvents();
+        this.prefillSearchInput();
+    }
+
+    bindEvents() {
+        this.search.addEventListener('input', (event) => {
+            this.clearAction.classList.toggle('hidden', !this.search.value);
```
Comment (vesan-83) [inline code/lightna/magento-os-frontend/js/common/Search.js:25]: make it extendable, onInput

File: code/lightna/magento-os-frontend/js/common/Search.js [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+        });
+
+        this.clearAction.addEventListener('click', (event) => {
+            this.search.focus();
+            this.clearAction.classList.add('hidden');
```
Comment (vesan-83) [inline code/lightna/magento-os-frontend/js/common/Search.js:30]: this.onClick

File: code/lightna/magento-os-frontend/layout/page.yaml [hunk 3]
```diff
@@ -32,8 +32,15 @@ template: page.phtml
       attributes:
         class:
           - header__container
-      .mobile-menu-toggle:
-        template: page/header/mobile-menu-toggle.phtml
+      .header-left:
```
Comment (vesan-83) [inline code/lightna/magento-os-frontend/layout/page.yaml:35]: it's already working a bit :+1: 
will suggestions also work? :)

Comment (vileyne) [reply code/lightna/magento-os-frontend/layout/page.yaml:35]: a bit? 😃 
can I show them by my own?

I am planning to add search history as next step / thinking about 'trending now' queries

Comment (vileyne) [reply code/lightna/magento-os-frontend/layout/page.yaml:35]: trending - I guess, can be info from Top Search Terms
history - no, I just liked in other shop and wanted to add, would you like me not too?

Comment (vileyne) [reply code/lightna/magento-os-frontend/layout/page.yaml:35]: yep, found it - `/search/ajax/suggest`
separate PR maybe?

Comment (vesan-83) [reply code/lightna/magento-os-frontend/layout/page.yaml:35]: up to you

Comment (vileyne) [reply code/lightna/magento-os-frontend/layout/page.yaml:35]: than let's merge search first and add suggestions after

**Review by vesan-83 — APPROVED** [review]

**Pull request merged**